loops_lab.js

```diff
@@ -0,0 +1,193 @@
+// # Loops Lab
```

Comment:
Good job overall with demonstration of for and while loops. With enough practice and repetition, using/understanding javascript will get easier. Keep up the good work and try bonuses.